public/react/components/Article.js

```diff
@@ -1,14 +1,13 @@
-import React from 'react';
+import React from 'react'
```

Comment:
srry this was just my prettier config

Reply:
haha thank you i can't beileve i missed the extra argument in viewPages